src/components/Heading/TypographyTypes.ts

```diff
@@ -0,0 +1,35 @@
+type ColorName = 'primary' | 'dark' | 'green';
```

Comment:
It would be nice to add both error (red) and warning (yellow). Of course, it depends on how we are going to implement it in the config file.